cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/spark/SparkCqlEvaluator.java

```diff
@@ -760,6 +777,29 @@ protected void writeResults(SparkSession spark, StructType schema, JavaPairRDD<O
                 .save(outputURI);
     }
 
+    private Map<String, String> createDataTypeAliases(List<ContextDefinition> filteredContexts, CqlToElmTranslator translator) {
+        Set<String> dataTypes = filteredContexts.stream().map(ContextDefinition::getPrimaryDataType).collect(Collectors.toSet());
+        Collection<ModelInfo> modelInfos = translator.getRegisteredModelInfos().values();
+
+        Map<String, String> dataTypeAliases = new HashMap<>();
+        for (ModelInfo modelInfo : modelInfos) {
+            modelInfo.getTypeInfo().stream().map(ClassInfo.class::cast)
```

Comment:
Can you guarantee that everything returned by `getTypeInfo()` can be cast to `ClassInfo`.
Would a `filter()` step for only objects that can't be cast be appropriate?